src/recipes/vscode/index.js

```diff
@@ -27,6 +28,26 @@ const getEdgeFunctionsPath = ({ config, repositoryRoot }) =>
 
 const getSettingsPath = (repositoryRoot) => join(repositoryRoot, '.vscode', 'settings.json')
 
+const hasDenoVSCodeExt = async () => {
+  const { stdout: extensions } = await execa('code', ['--list-extensions'], { stderr: 'inherit' })
+  return extensions.split('\n').includes('denoland.vscode-deno')
+}
+
+const getDenoVSCodeExt = async () => {
+  await execa('code', ['--install-extension', 'denoland.vscode-deno'], { stdio: 'inherit' })
+}
+
+const getDenoExtPrompt = () => {
+  const message = 'The Deno VSCode extension is recommended. Would you like to install it now?'
```

Comment:
I think this is the correct spelling, and what we've been using elsewhere in this file.

```suggestion
  const message = 'The Deno VS Code extension is recommended. Would you like to install it now?'
```